addons/advanced_throwing/functions/fnc_renderPickUpInteraction.sqf

```diff
@@ -31,7 +31,7 @@
             _nearThrowables append (ACE_player nearObjects ["ACE_Chemlight_IR_Dummy", PICK_UP_DISTANCE]);
 
             {
-                if (!(_x in _throwablesHelped) &&
+                if (!(_x in _throwablesHelped) && {!(_x getVariable [QGVAR(disablePickUp), false])} &&
```

Comment:
```suggestion
                if (!(_x in _throwablesHelped) && {(attachedTo _x) getVariable [QGVAR(allowPickup), true]} &&
```

Reply:
could put the var on the attached source, should still solve #8030

Reply:
I like this.

Couldn't decide between "disable" and "allow". I usually like the reverse (if disabled by default -> allow, if enabled by default -> disabled).

Reply:
```suggestion
                if (!(_x in _throwablesHelped) && {!((attachedTo _x) getVariable [QGVAR(disablePickUp), false])} &&
```

Reply:
otherway saves a `!` and `()` 😄

Reply:
Yeah :joy:  It of course doesn't matter at the end. Will update documentation. Did you test this?